libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestRequest.m

```diff
@@ -72,74 +70,6 @@ - (NSString *)description {
             ">",self, self.endpoint, methodName, self.path, paramStr];
 }
 
-- (SFRestAPISalesforceAction *)actionFromMethod:(SFRestMethod)method path:(NSString *)path {
```

Comment:
Removed a bunch of manipulative code from this file that's not required anymore.